vng_api_common/middleware.py

```diff
@@ -121,11 +123,15 @@ def applicaties(self) -> Union[list, None]:
 
     def _request_auth(self) -> list:
         client = AuthorizationsConfig.get_client()
+        try:
+            response = client.list(
+                'applicatie',
+                query_params={'client_ids': self.client_id}
+            )
+        except requests.exceptions.HTTPError:
```

Comment:
we don't want the underlying `requests` to leak - since it might be replaced with `aiohttp` for example. Probably better to catch `zds_client.ClientError`